### ISSUE TITLE
Cannot insert into table when statement contains output clause

### ISSUE THREAD

**Issue by lutzj82** [featurist/sworm#5]
I have a table with an identity column as well as a trigger on insert

```javascript
var group = db.model({
    table: 'tablename',
    id: 'identity_id'
});

var newGroup = group({
    column1: 'column1value',
    column2: 'column2value',
    column3: 96200
});

return newGroup.save();
```

The error I am getting is:

RequestError: The target table 'tablename' of the DML statement cannot have any enabled triggers if the statement contains an OUTPUT clause without INTO clause.

**Comment by refractalize** [comment]
hi @lutzj82, did you get this resolved? Which database are you using?

**Comment by lutzj82** [comment]
I have not been able to resolve, only workaround the issue. I can insert by using the query method, but that is not ideal. I am using MSSQL.

**Comment by refractalize** [comment]
Thanks @lutzj82, I've been able to reproduce the problem and make a fix in version 1.2.0.
